numpy/_core/code_generators/generate_numpy_api.py

```diff
@@ -41,18 +41,21 @@
 
 %s
 
+#include "numpy/utils.h"
+
 #if !defined(NO_IMPORT_ARRAY) && !defined(NO_IMPORT)
 static int
 _import_array(void)
 {
   int st;
-  PyObject *numpy = PyImport_ImportModule("numpy._core._multiarray_umath");
-  PyObject *c_api = NULL;
-
+  PyObject *numpy = _npy_import_numpy_multiarray_umath();
   if (numpy == NULL) {
+      PyErr_SetString(PyExc_ImportError,
+                      "_multiarray_umath failed to import");
```

Comment:
Do you need this, it should already say something similar?  But OK.

Generally seems fine, I am not immediately sure if `utils.h` is a good place for the helper.

The one worry I would have is that we might get a `ModuleNotFoundError` also because `_multiarray_umath` is missing (broken install).  However, not sure I want to worry too much about it, since I suspect we will have enough info available and `numpy` is probably more often imported in Python first anyway.